exercises/chapter 4/chapter4.Rmd

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Chapter 4: Workflows for R Coding'
+output:
+  html_document:
+    df_print: paged
+---
+
+# Exercise 1: Saving R Objects
```

Comment:
This is good. As an additional exercise, it would be good to have students then find the `my_vector.csv` they create and open in in RStudio, so that they can see what it looks like.